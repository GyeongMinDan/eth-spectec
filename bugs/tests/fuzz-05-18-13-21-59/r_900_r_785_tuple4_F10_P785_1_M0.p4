control c(out bit<16> r)() {
         apply {
          tuple<_, bit<16>> x = {
     10, 12 };
          if (((x) == ({
     10, 12 }
     ))) r = x[1];
        }
       }
