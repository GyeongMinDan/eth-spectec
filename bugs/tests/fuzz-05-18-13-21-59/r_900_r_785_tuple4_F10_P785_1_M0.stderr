likely_hits/r_900_r_785_tuple4_F10_P785_1_M0.p4(1): [--Wwarn=unused] warning: Control c is not used; removing
control c(out bit<16> r)() {
        ^
[--Wwarn=missing] warning: Program does not contain a `main' module
